packages/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -100,12 +101,129 @@ void main() {
     expect(() => controller.loadUrl('flutter.io'), throwsA(anything));
     expect(platformWebView.lastUrlLoaded, isNull);
   });
+
+  testWidgets('Check can go back', (WidgetTester tester) async {
+    WebViewController controller;
+    await tester.pumpWidget(
+      WebView(
+        onWebViewCreated: (WebViewController webViewController) {
+          controller = webViewController;
+        },
+      ),
+    );
+
+    expect(controller, isNotNull);
+
+    final bool canGoBackAnyPageLoaded = await controller.canGoBack;
```

Comment:
nit: you meant canGoBackNoPageLoaded?

Reply:
Sorry, I'm not good at English... -> dbe1b15